docs/modules/appinsights.md

```diff
@@ -0,0 +1,50 @@
+# [Azure Application Insights](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-overview)
+
+_Note: This module is EXPERIMENTAL. To enable this module, you must run Open Service Broker for Azure with modules.minStability set to `experimental`_
+
+## Services & Plans
+
+### Service: azure-appinsights
+
+| Plan Name | Description |
+|-----------|-------------|
+| `default` | For general application. |
```

Comment:
Noticed there are several types of application, why only `general` is added?

Reply:
Ah, I will add others as plans.